src/util/Handlers.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) 2018-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['util/Logger'], function (Logger) {
+
+  var Handlers = {};
+
+  Handlers.defaultSuccessTokenHandler = function(tokenManager, keys = {}) {
+    /**
+     * Default success handler which is normally invoked when
+     * `parseAndStoreTokenFromUrl` is called. This success handler will
+     * automatically store the tokens with the keys "accessToken" and
+     * "idToken", if returned in the response.
+     */
+
+    var tokenKeys = {
+      accessToken: keys.ACCESS_TOKEN || 'accessToken',
+      idToken: keys.ID_TOKEN || 'idToken'
+    };
+
+    return function (res) {
+      var tokens = Array.isArray(res) ? res : [res];
+      for (var i = 0; i < tokens.length; ++i) {
+        if (tokens[i].idToken) {
+          tokenManager.add(tokenKeys.idToken, tokens[i]);
+        } else if (tokens[i].accessToken) {
+          tokenManager.add(tokenKeys.accessToken, tokens[i]);
+        }
+      }
+    };
+  };
+
+  Handlers.defaultErrorHandler = function(err) {
```

Comment:
Can we reuse this for what we added in https://github.com/okta/okta-signin-widget/pull/466?

Reply:
Internally, yes. This would apply to any error handler currently with the Widget. In #466, we [hardcoded the default handler](https://github.com/okta/okta-signin-widget/pull/466/files#diff-d57a8731b033a3e50aa86ec27435a102R85) to ensure a global change in the function wouldn't cause unexpected changes (i.e change from `.error` to `.warn`).

The previous design pattern forced developers to include their own `success` and `error` handlers, where this new approach (BYO or use ours) provides developers with significantly more flexibility.